pkg/client/clientset/versioned/typed/shipper/v1alpha1/fake/fake_rolloutblock.go

```diff
@@ -0,0 +1,124 @@
+	v1alpha1 "github.com/bookingcom/shipper/pkg/apis/shipper/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	labels "k8s.io/apimachinery/pkg/labels"
+	schema "k8s.io/apimachinery/pkg/runtime/schema"
+	types "k8s.io/apimachinery/pkg/types"
+	watch "k8s.io/apimachinery/pkg/watch"
+	testing "k8s.io/client-go/testing"
+)
+
+// FakeRolloutBlocks implements RolloutBlockInterface
+type FakeRolloutBlocks struct {
+	Fake *FakeShipperV1alpha1
+	ns   string
+}
+
+var rolloutblocksResource = schema.GroupVersionResource{Group: "shipper.booking.com", Version: "v1alpha1", Resource: "rolloutblocks"}
+
+var rolloutblocksKind = schema.GroupVersionKind{Group: "shipper.booking.com", Version: "v1alpha1", Kind: "RolloutBlock"}
+
+// Get takes name of the rolloutBlock, and returns the corresponding rolloutBlock object, and an error if there is any.
+func (c *FakeRolloutBlocks) Get(name string, options v1.GetOptions) (result *v1alpha1.RolloutBlock, err error) {
+	obj, err := c.Fake.
+		Invokes(testing.NewGetAction(rolloutblocksResource, c.ns, name), &v1alpha1.RolloutBlock{})
+
+	if obj == nil {
+		return nil, err
+	}
+	return obj.(*v1alpha1.RolloutBlock), err
+}
+
+// List takes label and field selectors, and returns the list of RolloutBlocks that match those selectors.
+func (c *FakeRolloutBlocks) List(opts v1.ListOptions) (result *v1alpha1.RolloutBlockList, err error) {
+	obj, err := c.Fake.
+		Invokes(testing.NewListAction(rolloutblocksResource, rolloutblocksKind, c.ns, opts), &v1alpha1.RolloutBlockList{})
+
+	if obj == nil {
+		return nil, err
+	}
+
+	label, _, _ := testing.ExtractFromListOptions(opts)
+	if label == nil {
+		label = labels.Everything()
+	}
+	list := &v1alpha1.RolloutBlockList{ListMeta: obj.(*v1alpha1.RolloutBlockList).ListMeta}
+	for _, item := range obj.(*v1alpha1.RolloutBlockList).Items {
+		if label.Matches(labels.Set(item.Labels)) {
+			list.Items = append(list.Items, item)
+		}
+	}
+	return list, err
+}
+
+// Watch returns a watch.Interface that watches the requested rolloutBlocks.
+func (c *FakeRolloutBlocks) Watch(opts v1.ListOptions) (watch.Interface, error) {
+	return c.Fake.
+		InvokesWatch(testing.NewWatchAction(rolloutblocksResource, c.ns, opts))
+
+}
+
+// Create takes the representation of a rolloutBlock and creates it.  Returns the server's representation of the rolloutBlock, and an error, if there is any.
+func (c *FakeRolloutBlocks) Create(rolloutBlock *v1alpha1.RolloutBlock) (result *v1alpha1.RolloutBlock, err error) {
+	obj, err := c.Fake.
+		Invokes(testing.NewCreateAction(rolloutblocksResource, c.ns, rolloutBlock), &v1alpha1.RolloutBlock{})
+
+	if obj == nil {
+		return nil, err
+	}
+	return obj.(*v1alpha1.RolloutBlock), err
+}
+
+// Update takes the representation of a rolloutBlock and updates it. Returns the server's representation of the rolloutBlock, and an error, if there is any.
+func (c *FakeRolloutBlocks) Update(rolloutBlock *v1alpha1.RolloutBlock) (result *v1alpha1.RolloutBlock, err error) {
+	obj, err := c.Fake.
```

Comment:
Not sure about this line break. Let's' keep method invocations on the same line.